protocol/messenger_status_updates.go

```diff
@@ -267,3 +267,59 @@ func (m *Messenger) HandleStatusUpdate(state *ReceivedMessageState, statusMessag
 func (m *Messenger) StatusUpdates() ([]UserStatus, error) {
 	return m.persistence.StatusUpdates()
 }
+
+func (m *Messenger) timeoutStatusUpdates(fromClock uint64, tillClock uint64) {
+	// Most of the time we only need to time out just one status update,
+	// but the range covers special cases like, other status updates had the same clock value
+	// or the received another status update with higher clock value than the reference clock but
+	// lower clock value than the statusUpdate.Clock
+	deactivatedStatusUpdates, err := m.persistence.DeactivatedAutomaticStatusUpdates(fromClock, tillClock)
+
+	// Send deactivatedStatusUpdates to Client
+	if err == nil {
+		m.config.messengerSignalsHandler.StatusUpdatesTimedOut(&deactivatedStatusUpdates)
+	} else {
+		m.logger.Debug("Unable to deactivate atutomatic status updates", zap.Error(err))
```

Comment:
atutomatic  -> automatic, rather "Unable to fetch/get deact...." because you read from db and not deactivate, right ?

Reply:
Fixed typo and debug log